src/main/java/com/google/sps/data/Project.java

```diff
@@ -0,0 +1,54 @@
+package com.google.sps.data;
+
+/** Class containing projects*/
```

Comment:
What's a project?

Reply:
The API we're using returns a list of projects or initiatives that different non-profits are doing. So a project is meant to represent those.